NOTES-WINDOWS.md

```diff
@@ -125,6 +125,31 @@ Administrator" before running `nmake install`.  The other solution
 is, of course, to choose a different set of directories by using
 `--prefix` and `--openssldir` when configuring.
 
+Note that, on Windows platforms (both 32 and 64 bit), the above build time
+defaults can be overridden by registry keys.  This is done because it is common
+practice for windows based installers to allow users to place the installation
+tree at various locations not defined at build time.  The following keys:
+
+    `\\HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\OpenSSL-<version>-<ctx>\OPENSSLDIR`
+    `\\HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\OpenSSL-<version>-<ctx>\ENGINESDIR`
+    `\\HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\OpenSSL-<version>-<ctx>\MODULESDIR`
```

Comment:
yes, reg.exe inserts the Wow6432Node element into the path when /reg:32 or /reg:64 is specified.

However, when looking at the key paths with regedit, the node is explicitly displayed, so I thought it would be better to explicitly include it here so it visually matched up to an adminstrator.